src/Playwright.TestingHarnessTest/playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  timeout: 2 * 60 * 1_000,
+  workers: 1,
+  reporter: 'list',
```

Comment:
Do you want `list` in CI, or `dot`?

Reply:
since we have little tests which take long, list is helpful there!